personalize.admin.campaign.inc

```diff
@@ -177,6 +171,12 @@ function personalize_campaign_wizard_process_bar($form, &$form_state, $agent_dat
       'wrapper' => 'personalize-campaign-wizard-process-bar',
       'effect' => 'fade',
     );
+
+    // Save the current subform.
+    $form['process_bar']['actions']['save'] = array(
+      '#type' => 'submit',
+      '#value' => t('Save'),
+    );
   }
 
```

Comment:
Rotated the "status" button and the "save" button sequence, because the status button can now possibly be a "Review & save" link, which looks nicer when placed before the "save" button.